base/sparse/sparsematrix.jl

```diff
@@ -2936,11 +2920,11 @@ function blkdiag(X::SparseMatrixCSC...)
 end
 
 ## Structure query functions
-issymmetric(A::SparseMatrixCSC) = is_hermsym(A, IdFun())
+issymmetric(A::SparseMatrixCSC) = is_hermsym(A, identity)
 
-ishermitian(A::SparseMatrixCSC) = is_hermsym(A, ConjFun())
+ishermitian(A::SparseMatrixCSC) = is_hermsym(A, conj)
 
-function is_hermsym(A::SparseMatrixCSC, check::Func)
+function is_hermsym(A::SparseMatrixCSC, check::Function)
```

Comment:
True, `Callable` (as an alias for `Union{Function,DataType}` that could easily be changed in the future) would probably be better, but `Function` is used all over the place in signatures. Maybe better handled in separate PR?